sycl/test-e2e/Matrix/joint_matrix_apply_cuda.hpp

```diff
@@ -8,14 +8,10 @@
 
 // cuda backend test for joint_matrix_apply (and joint_matrix_fill)
 
-#include <sycl/sycl.hpp>
+#include "common.hpp"
```

Comment:
If you are changing tests for CUDA, please, check that they are passing on CUDA after your change.

Reply:
Passed see below.

Reply:
I don't see this test running at all